packages/core-mobile/app/hooks/networks/useNetworksListener.ts

```diff
@@ -14,7 +13,7 @@ export const useNetworksListener = (): void => {
   useEffect(() => {
     return dispatch(
       addListener({
-        matcher: isAnyOf(toggleDeveloperMode, onAppUnlocked),
+        matcher: isAnyOf(toggleDeveloperMode),
```

Comment:
this was dispatching a networksFetched action on app unlocked that triggered a balance refetch. as a result we were fetching balance twice on app unlocked, which took a lot of the js thread.